src/ResourceManager/Sql/Commands.Sql/packages.config

```diff
@@ -1,5 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.Sql" version="1.10.0-preview" targetFramework="net452" />
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Sql" version="1.13.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="2.4.0-preview" targetFramework="net45" />
+  <package id="WindowsAzure.Storage" version="8.1.1" targetFramework="net452" />
 </packages>
```

Comment:
remove the two files below - these are changed during build but should not be committed.